### PR TITLE
[WIP] update to homeBeach.js function

### PR DESCRIPTION
**Pull request by focused220**
Currently this function receives a complete user object of the following format: 
const user = {
    cognitoUserId,
    fullName,
    email,
    homeBeach,
    homeBeachName,
    longitude,
    latitude
}

Once received, this function will query the current active DB for the closest beach location and return a complete location object. 

Setting homeBeachName on the user object pending.

### CODE REVIEW

**Comment by focused220** [comment]
Will be ready for merge once tied in to GraphQL

**Comment by jthicks91** [comment]
Logic makes sense and looks good. Just need to be integrated with GraphQL.

File: schema/schema.js [hunk 1]
```diff
@@ -192,6 +192,16 @@ const RootQuery = new GraphQLObjectType({
         return User.findById(args.id);
       }
     },
+    filterUser: {
```
Comment (jthicks91) [inline schema/schema.js]: This is great Anthony, nice work!

**Comment by stackBlock** [comment]
FYI - please use ``` filterUser ``` to search for users the same way you would search through locations. that has been updated and pushed to the feature branch.

**Comment by stackBlock** [comment]
RESTROOM was set to number. Changed to String in location.js file.

**Pull request merged**